docs/tutorial/macos-dock.md

```diff
@@ -1,10 +1,11 @@
 # macOS Dock
 
+## Overview
+
 Electron has APIs to configure the app's icon in the macOS Dock. A macOS-only
-API exists to create a custom dock menu, but
-Electron also uses the app's dock icon to implement cross-platform features
-like [recent documents][recent-documents] and
-[application progress][progress-bar].
+API exists to create a custom dock menu, but Electron also uses the app's dock
+icon to implement cross-platform features like
+[recent documents][recent-documents] and [application progress][progress-bar].
```

Comment:
Unrelated to your change, but I'd say we should change the wording here because I feel like _implementing_ is the wrong word here.

```suggestion
API exists to create a custom dock menu, but Electron also uses the app dock
icon as the entry point for cross-platform features like
[recent documents][recent-documents] and [application progress][progress-bar].
```